packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# 49.10.0 - September 16, 2020
+
+## Fixed
+
+#### salesforcedx-vscode-soql
+
+- Fix a compile error by adding type for mkdirp ([PR #2524](https://github.com/forcedotcom/salesforcedx-vscode/pull/2524))
+
+## Added
+
+#### salesforcedx-vscode-soql
+
+- Soql Builder - Transform soql to ui and back to soql ([PR #2501](https://github.com/forcedotcom/salesforcedx-vscode/pull/2501))
```

Comment:
Do we know if the SOQL Builder is ready for users yet? My guess is no and that we might not want to include these PRs in the change log yet.
@lcampos

Reply:
That is correct, we should not add anything related to soql until end of October